src/blocks/Promo/Promo.js

```diff
@@ -64,22 +61,24 @@ function Promo(props) {
               <Typography>{description}</Typography>
             </CardContent>
           </Box>
-          {alignContent === 'left' && (
-            <CardMedia
-              component="img"
-              image={imageSrc}
-              alt={name}
-              sx={{
-                width: '300px',
-                clipPath: () => {
-                  if (imageCircle) {
-                    return 'circle(100px at 50%)'
-                  }
-                  return 0
-                },
-              }}
-            />
-          )}
+          <Media
+            {...(mediaProps?.component === 'video'
+              ? {
+                  autoPlay: true,
+                  muted: true,
+                  loop: true,
+                  playsInline: true,
+                }
+              : { alt: { name } })}
+            {...mediaProps}
+            sx={{
+              clipPath: imageCircle ? `circle(${circleSettings})` : 0,
+              '@media(max-width: 900px)': {
```

Comment:
please use the theme based breakpoints: [theme.breakpoints.up('SCREENSIZE')]: { ................. },

instead of max-width